pom.xml

```diff
@@ -4400,12 +4399,6 @@
         <version>${version.org.yaml.snakeyaml}</version>
       </dependency>
 
-      <dependency>
```

Comment:
why are you removing the dependency ?

Reply:
there were 2 postgresql dependencies, one older with `postgresql` groupId and the newer which is latest one with `org.postgresql` groupId